shotgun_api3/shotgun.py

```diff
@@ -593,6 +599,17 @@ def find(self, entity_type, filters, fields=None, order=None,
         :param include_archived_projects: Optional, flag to include entities
         whose projects have been archived
 
+        :param additional_filter_presets: Optional list of presets to
+        further filter the result set, list has the form:
+        [ { preset_name: <preset_name>, <optional_preset parameters> } ],
```

Comment:
For python it should be something more like this for clarity and the fact that keys need quotes:
[{"preset_name": <preset_name>, <optional_param1>: <optional_value1>, ... }]